keyboards/papercranekeyboards/gerald65/config.h

```diff
@@ -0,0 +1,138 @@
+// Copyright 2021 PaperCraneKeyboards (@PaperCraneKeyboards)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x1003 // P,C (Paper Crane)
```

Comment:
https://www.the-sz.com/products/usbid/index.php?v=0x1003&p=&n=

This Vendor Id is in use, and needs to be changed. 

Anything above 0x4000 is relatively safe.

Reply:
Oh thank you! Changed to 0x5012.